src/test/groovy/com/ss/mqtt/broker/test/integration/ConnectionTest.groovy

```diff
@@ -25,6 +40,33 @@ class ConnectionTest extends MqttBrokerTest {
             !result.sessionPresent
     }
     
+    def "subscriber should connect to broker with user and pass"() {
+        given:
+            def client = MqttClient.builder()
```

Comment:
you can add factory method to create a mqtt client to the base class of integration tests